docs/pipeline/actions/attributes/piimasking.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: "Pii Masking"
+sidebarTitle: "Pii Masking"
+---
+
+The "PII Masking" Odigos Action can be used to mask PII data from span attribute values.
+
+### Use Cases
+
+#### Security
+
+- By default, OpenTelemetry should not record PII (Personally Identifiable Information) or sensitive data such as passwords, api tokens, etc. However, if the data is accidentally added to the telemetry signals, or if a manual instrumentation recorded it into an attribute, this action can be used to mask such data.
+
+#### Legal and Compliance
+
+- Ensure compliance with legal and privacy.
+    - EU General Data Protection Regulation (GDPR) prohibits txhe transfer of any personal data like birthdates, addresses, or ip addresses across borders without explicit consent from the data subject.
+    - Health Insurance Portability and Accountability Act (HIPAA) requires that all health data be protected from unauthorized access.
+    - Payment Card Industry (PCI) Data Security Standards prohibit logging certain things or storing them unencrypted.
+
+
+### Mechanism
+
+This action processes a list of PII categories to be masked from the traces. Each PII category is linked to a regex expression that will be applied to all span attribute values. Any portion of the span attribute values that match the regex expression will be replaced with `****`, while the rest of the attribute value will remain unchanged.
+
+
+### Basic Example
+
+The basic example below shows how to configure the Pii Masking action to mask Visa and Master cards from span attribute values.
+
+
+Create a file named `piimasking.yaml` with the following content:
+
+```yaml
+apiVersion: actions.odigos.io/v1alpha1
+kind: PiiMasking
+metadata:
+  name: piimasking
+  namespace: odigos-system
+spec:
+  actionName: "PII Masking"
+  piiCategoryConfigurations:
+    - mask: true
+      category: CREDIT_CARD
+  signals:
+    - TRACES
+```
+
+Apply the action to the cluster:
+
+```bash
+kubectl apply -f piimasking.yaml
+```
+
+### Full Action Options
+
+The full list of options available for the "PiiMasking" action are:
+
+- `piiCategoryConfigurations` (required): An array of dictionaries where each dictionary has two keys: `mask` and `category`. `category` is the PII Category you want to mask and `mask` is a boolean value that indicates if to mask or not.
```

Comment:
I wonder if we should also document the supported values for `category`. How can a user discover the list value to populate here?